setup.py

```diff
@@ -25,9 +25,13 @@ def readme():
           'pyhocon',
           'mock',
           'arrow',
-          'nose'
+          'nose',
+          'requests',
+          'thrift==1.0.0-py3',
+          'munch'
       ],
       entry_points={'console_scripts': ['herc = herc.webservice:main']},
+      dependency_links = [urllib.parse.urljoin('file:', os.path.join(os.getcwd(), 'thrift-1.0.0-py3.tar.gz'))],
```

Comment:
That tarball is already getting copied to the appropriate place - is there another reason why we should add it to package_data?